raw_data/preprocessed/europe/france_french.txt

```diff
@@ -0,0 +1,97 @@
+
+(2) Les travaux et projets d'aménagement qui sont entrepris par une collectivité publique ou qui nécessitent une autorisation ou une décision d'approbation ainsi que les documents d'urbanisme doivent respecter les préoccupations d'environnement. Les études préalables à la réalisation d'aménagements ou d'ouvrages qui, par l'importance de leurs dimensions ou leurs incidences sur le milieu naturel, peuvent porter atteinte à ce dernier, doivent comporter une étude d'impact permettant d'en apprécier les conséquences. Un décret en Conseil d'État précise les modalités d'application du présent article. Il fixe notamment: d'une part, les conditions dans lesquelles les préoccupations d'environnement sont prises en compte dans les procédures réglementaires existantes; d'autre part: le contenu de l'étude d'impact qui comprend au minimum une analyse de l'état initial du site et de son environnement, l'étude des modifications que le projet y engendrerait et les mesures envisagées pour supprimer, réduire et, si possible, compenser les conséquences dommageable pour l'environnement; les conditions dans lesquelles l'étude d'impact sera rendue publique; la liste limitative des ouvrages qui, en raison de la faiblesse de leurs répercussions sur l'environnement, ne sont pas soumis à la procédure de l'étude d'impact. Il fixe également les conditions dans lesquelles le ministre chargé de l'environnement pourra se saisir ou être saisi, pour avis, de toute étude d'impact. Si une requête déposée devant la juridiction administrative contre une autorisation ou une décision d'approbation d'un projet visé à l'alinéa premier du présent article est fondée sur l'absence d'étude d'impact, la juridiction saisie fait droit à la demande de sursis à exécution de la décision attaquée dès que cette absence est constatée selon une procédure d'urgence.
+
+CHAPITRE I - De la protection de la faune et de la flore
+
+(3) Lorsqu'un intérêt scientifique particulier ou que les nécessités de la préservation du patrimoine biologique national justifient la conservation d'espèces animales non domestiques ou végétales non cultivées, sont interdits: la destruction ou l'enlèvement des œufs ou des nids, Ia mutilation, la destruction, la capture ou l'enlèvement, la naturalisation d'animaux de ces espèces ou, qu'ils soient vivants ou morts, leur transport, leur colportage, leur utilisation, leur mise en vente, leur vente ou leur achat; la destruction, la coupe, la mutilation, l'arrachage, la cueillette ou l'enlèvement de végétaux de ces espèces, ou de leurs fructifications, leur transport, leur colportage, leur utilisation, leur mise en vente, leur vente ou leur achat; la destruction, l'altération ou la dégradation du milieu particulier à ces espèces animales ou végétales; la destruction des sites contenant des fossiles permettant d'étudier l'histoire du monde vivant ainsi que les premières activités humaines.
```

Comment:
I believe the "Ia" preceding "mutilation" should be LA and not IA (capitals used only to show the difference between L and I).

Reply:
Both letters should be lowercase. Capital eye looks like a lowercase el on Google docs. I will update.

Reply:
I will also replace `œufs` with `oeufs`. I created [this issue](https://github.com/danpdraper/eia/issues/90) to ensure that all instances of `œ` in our body of legislation are replaced with `oe`.

Reply:
Ok, thanks for letting me know.